libs/payments/paypal/src/lib/client.spec.ts

```diff
@@ -366,4 +363,63 @@ describe('PayPalClient', () => {
       expect(result).toMatchObject(response);
     });
   });
+
+  describe('getListOfPayPalClientErrors', () => {
+    const raw = faker.word.words();
+
+    it('returns a general error message when no errors are provided', () => {
+      const data = NVPErrorResponseFactory({ L: undefined });
+      const actual = client.getListOfPayPalClientErrors(raw, data);
+      const expected = [
+        new PayPalClientError(raw, data, {
+          message:
+            'PayPal NVP returned a non-success ACK. See "this.raw" or "this.data" for more details.',
+        }),
+      ];
+      expect(actual).toStrictEqual(expected);
+    });
+
+    it('handles multiple errors', () => {
+      const err1 = NVPErrorFactory();
+      const err2 = NVPErrorFactory();
+      const data = NVPErrorResponseFactory({
+        L: [err1, err2],
+      });
+      const actual = client.getListOfPayPalClientErrors(raw, data);
+      const expected = [
+        new PayPalClientError(raw, data, {
+          message: err1.LONGMESSAGE,
+          errorCode: parseInt(err1.ERRORCODE),
```

Comment:
Not sure if we have the rules enabled, but ESLint generally recommends adding a base-10 to `parseInt()` to avoid weird hex edge cases.
```suggestion
          errorCode: parseInt(err1.ERRORCODE, 10),
```

But it's also just a *.spec.ts file, so 🤷

Reply:
Not a concern in this case.